src/vue/fields/TickField.vue

```diff
@@ -135,6 +132,7 @@ export default {
   position: relative;
   display: flex;
   width: fit-content;
+  padding-top: 1.5rem;
```

Comment:
the tick-field by itself should not be responsible for itself position related to other elements of the page. only elements of the tick-field could be arranged inside the tick-field